src/osd/osd_types.h

```diff
@@ -2962,6 +2962,11 @@ struct pg_log_t {
   mempool::osd::list<pg_log_entry_t> log;  // the actual log.
   
   pg_log_t() = default;
+  pg_log_t(const pg_log_t& other) = default;
+  pg_log_t& operator=(const pg_log_t& other) = default;
+  pg_log_t(pg_log_t&& other) = default;
+  pg_log_t& operator=(pg_log_t&& other) = default;
+
```

Comment:
because pg_log_t() has a non-default constructor. that prevent the compiler from generating these. that's also why we put
```c++
pg_log_t() = default;
```

for the default ctor.